frontend/src/firebase/actions.ts

```diff
@@ -425,6 +431,30 @@ export function completeTaskInFocus<T extends { readonly id: string; readonly or
  * @return a new list with updated orders.
  */
 export function applyReorder(orderFor: 'tags' | 'tasks', reorderMap: Map<string, number>): void {
+  const { tags, tasks } = store.getState();
+  if (orderFor === 'tags') {
+    const editedTags: Tag[] = [];
+    Array.from(reorderMap.entries()).forEach(([id, order]) => {
+      const existingTag = tags.get(id);
+      if (existingTag != null) {
```

Comment:
Since `.get()` in the tags map can only return the value or `undefined` at id in the function call above this line, can't this be refactored this be `existingTag !== undefined` instead?

Reply:
Changed